### PR TITLE
Pomodoro

### PR DESCRIPTION
**Pull request by hrishikeshmane**
Change Logs-

1. Pomodoro feature added
2. MUI Theme Integration
3. Context API for Todo lists
4. Persistent storage of todos

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,8 +1,40 @@
...
+import { PomodoroProvider } from "./context/pomodoro-context";
 
 function App() {
-  return <div className="App"> </div>;
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    let authToken = sessionStorage.getItem("Auth Token");
+
+    if (!authToken) {
```
Comment (SanketDhabarde) [inline src/App.tsx:18]: Suggestion: You can use `RequiresAuth` and redirect the user if it is not logged in

File: src/util/index.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+function getFromStorage(key: string, type: "local" | "session", fallback: any) {
```
Comment (SanketDhabarde) [inline src/util/index.ts:1]: Liked this util method

**Review by SanketDhabarde — APPROVED** [review]
- Good folder structure
- Code looks clean
- You can add screenshots/gifs/previews to show what changes you made

**Review by jeetbhalani15 — APPROVED** [review]
- context works fine 
- code looks good
 - nice UI

**Review by aniketsingh13 — APPROVED** [review]
code is easy to read
file structure is good
good to merge

**Pull request merged**